.eslintrc

```diff
@@ -88,9 +89,11 @@
             "files": ["**/test/**/*.js", "**/test/**/*.ts", "**/test/**/*.tsx"],
             "rules": {
                 "no-console": "off",
+                "jsdoc/require-jsdoc": "off",
                 "jsdoc/require-returns-description": "off",
                 "jsdoc/require-param-description": "off",
-                "max-nested-callbacks": ["warn", 5]
+                "max-nested-callbacks": ["warn", 5],
+                "sonarjs/cognitive-complexity": "off"
```

Comment:
How did this one `"sonarjs/cognitive-complexity": "off"` sneak in???

Reply:
This is only for the test files. There were some tests where this was failing and I think it should be OK for this to be turned off for tests.